ssas/service/admin/api_test.go

```diff
@@ -0,0 +1,71 @@
+package admin
+
+import (
+	"encoding/json"
+	"github.com/CMSgov/bcda-app/bcda/database"
+	"github.com/CMSgov/bcda-app/ssas"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+type APITestSuite struct {
+	suite.Suite
+	db *gorm.DB
+}
+
+func (s *APITestSuite) SetupSuite() {
+	s.db = database.GetGORMDbConnection()
```

Comment:
this is now an exported func in the `ssas` package so we should use that one to keep it independent